docs/test-pages/TCD-009.adoc

```diff
@@ -0,0 +1,46 @@
+= TCD-009
+
+== Purpose
+
+Verifying the primitive types and their behaviour. This case is verifying the optional primitive fields by adding values. This is a simple happy path.
```

Comment:
Something does not add up here. Why are we testing optional attributes if we are assigning values anyway?

If this was a reverse engineered test, then it must have been made to verify whether DEFAULT VALUES are ignored and initial values are used. This is only a guess. So either way, the "Purpose" section should be clarified.